src/odemis/driver/xt_client.py

```diff
@@ -2147,10 +2166,17 @@ class Stage(model.Actuator):
     moving the TFS stage and updating the position.
     """
 
-    def __init__(self, name: str, role: str, parent: model.HwComponent, rng: dict = None, **kwargs) -> None:
+    def __init__(self, name: str, role: str, parent: model.HwComponent, rng: dict = None, raw: bool = False,
+                 **kwargs) -> None:
```

Comment:
This needs a docstring. At least it should document the new "raw" argument. It'd be a good opportunity to document rng too.